site/en/codelabs/openthread-border-router-nat64/index.lab.md

```diff
@@ -0,0 +1,377 @@
+---
+id: openthread-border-router-nat64
+summary: In this codelab, you'll build an OpenThread border router with NAT64 support, and use the end-device in the network to access IPv4 only resources from the internet.
+status: [final]
+authors: Song Guo
+categories: Nest
+feedback_link: https://github.com/openthread/ot-br-posix/issues
+keywords: openthread, nat64
+project: /_project.yaml
+book: /_book.yaml
+
+---
+
+# Thread Border Router - Provide Internet access via NAT64
+
+[Codelab Feedback](https://github.com/openthread/ot-br-posix/issues)
+
+
+## Introduction
+
+
+
+<img src="img/7299534792dd9439.png" alt="7299534792dd9439.png"  width="624.00" />
+
+### What is Thread?
+
+Thread is an IP-based low-power wireless mesh networking protocol that enables secure device-to-device and device-to-cloud communications. Thread networks can adapt to topology changes to avoid single-point failures.
+
+### What is OpenThread?
+
+[OpenThread released by Google](https://github.com/openthread/openthread) is an open-source implementation of Thread®.
+
+### **What is an OpenThread Border Router?**
```

Comment:
Can you remove the bold from all the headers throughout the codelab?  We need to clean that up in the other codelabs, but let's go ahead and remove it from here now please.